src/SFML/Window/Context.cpp

```diff
@@ -49,19 +49,40 @@ namespace sf
 Context::Context() : m_context(priv::GlContext::create())
 {
     if (!setActive(true))
-    {
         err() << "Failed to set context as active during construction" << std::endl;
-    }
 }
 
 
 ////////////////////////////////////////////////////////////
 Context::~Context()
 {
-    if (!setActive(false))
-    {
+    if (m_context && !setActive(false))
         err() << "Failed to set context as inactive during destruction" << std::endl;
-    }
+}
+
+
+////////////////////////////////////////////////////////////
+Context::Context(Context&& other) noexcept : m_context(std::move(other.m_context))
+{
+    if (!setActive(true))
+        err() << "Failed to set context as active during construction" << std::endl;
+}
+
+
+////////////////////////////////////////////////////////////
+Context& Context::operator=(Context&& other) noexcept
+{
+    if (this == &other)
+        return *this;
+
+    if (m_context && !setActive(false))
+        err() << "Failed to set context as inactive during move assignment" << std::endl;
+
+    m_context = std::move(other.m_context);
+    if (m_context && !setActive(true))
+        err() << "Failed to set context as active during construction" << std::endl;
```

Comment:
Copy-paste detected :nerd_face:
```suggestion
        err() << "Failed to set context as active during move assignment" << std::endl;
```

Reply:
😰